Comments/ClientApp/src/components/SubmittedContent/__tests__/SubmittedContent.test.js

```diff
@@ -0,0 +1,115 @@
+			isOrganisationCommenter: false,
+			isLead: false,
+			consultationState:{
+				supportsDownload: true,
+				leadHasBeenSentResponse: false,
+			},
+			consultationId: 1,
+			basename: "something",
+			isSubmitted: true,
+			linkToReviewPage: false,
+		};
+
+        it("should match snapshot for individual commenters on the review page", () => {
+			const wrapper = shallow(<SubmittedContent {...fakeProps} />);
+
+			expect(
+				toJson(wrapper, {
+					noKey: true,
+					mode: "deep",
+				})).toMatchSnapshot();
+		});
+
+		it("should match snapshot for individual commenters on the submitted page", () => {
+			const localProps = fakeProps;
+			localProps.linkToReviewPage = true;
+			const wrapper = shallow(<SubmittedContent {...localProps} />);
+
+			expect(
+				toJson(wrapper, {
+					noKey: true,
+					mode: "deep",
+				})).toMatchSnapshot();
+		});
+
+		it("should match snapshot for organisation commenter on the review page", () => {
+			const localProps = fakeProps;
+			localProps.isOrganisationCommenter = true;
+            localProps.linkToReviewPage = false;
+			const wrapper = shallow(<SubmittedContent {...localProps} />);
+
+			expect(
+				toJson(wrapper, {
+					noKey: true,
+					mode: "deep",
+				})).toMatchSnapshot();
+		});
+
+		it("should match snapshot for organisation commenter on the submitted page", () => {
+			const localProps = fakeProps;
+			localProps.isOrganisationCommenter = true;
+			localProps.linkToReviewPage = true;
+			const wrapper = shallow(<SubmittedContent {...localProps} />);
+
+			expect(
+				toJson(wrapper, {
+					noKey: true,
+					mode: "deep",
+				})).toMatchSnapshot();
+		});
+
+		it("should match snapshot for organisation lead on the review page without any responses from their org", () => {
+			const localProps = fakeProps;
+			localProps.isLead = true;
+            localProps.linkToReviewPage = false;
+			const wrapper = shallow(<SubmittedContent {...localProps} />);
+
+			expect(
+				toJson(wrapper, {
+					noKey: true,
+					mode: "deep",
+				})).toMatchSnapshot();
+		});
+
+		it("should match snapshot for organisation lead on the review page with responses from their org", () => {
+			const localProps = fakeProps;
+			localProps.isLead = true;
+			localProps.consultationState.leadHasBeenSentResponse = true;
+            localProps.linkToReviewPage = false;
+			const wrapper = shallow(<SubmittedContent {...localProps} />);
+
+			expect(
+				toJson(wrapper, {
+					noKey: true,
+					mode: "deep",
+				})).toMatchSnapshot();
+		});
+
+		it("should match snapshot for organisation lead on the submitted page", () => {
+			const localProps = fakeProps;
+			localProps.isLead = true;
+			localProps.linkToReviewPage = true;
+			localProps.consultationState.leadHasBeenSentResponse = true;
+			const wrapper = shallow(<SubmittedContent {...localProps} />);
+
+			expect(
+				toJson(wrapper, {
+					noKey: true,
+					mode: "deep",
+				})).toMatchSnapshot();
+		});
+
+	});
+});
```

Comment:
Could the test names be a bit clearer? I think mentioning the review page and the submitted page suggests that you're snapshotting those components rather than just the SubmittedContent component with the appropriate props.